.env

```diff
@@ -0,0 +1 @@
+BROWSER=none
```

Comment:
This is not too important for now, but notice how GitHub provides a warning symbol when there is no newline at EOF. It's a [UNIX thing](https://thoughtbot.com/blog/no-newline-at-end-of-file#history-lesson). You should configure your editor to automatically insert newlines when appropriate.

Reply:
> Related to the `webpack` comment, we should probably also delete `webpack.config.js` as part of this change.
> 
> You can amend the existing commit after making the requested changes.

Ok, I forget that, webpack.config is now removed.

Reply:
> This is not too important for now, but notice how GitHub provides a warning symbol when there is no newline at EOF. It's a [UNIX thing](https://thoughtbot.com/blog/no-newline-at-end-of-file#history-lesson). You should configure your editor to automatically insert newlines when appropriate.

This is fixed also, I configured VS code to add new line on save.